.eslintrc.cjs

```diff
@@ -145,9 +145,10 @@ const rules = {
       'allowAllPropertiesOnSameLine': true,
     },
   ],
-  'operator-linebreak': 'error',
+  'operator-linebreak': 'off',
```

Comment:
This is the old behavior, rather than just disabling this.

```suggestion
  'operator-linebreak': [
    'error',
    'after',
    {
      'overrides': {
        ':': 'before',
        '?': 'before',
      },
    },
  ],
```